### DISCUSSION TITLE
prisma init projectName not generating anything

### DISCUSSION THREAD

**Discussion by JohnFajardo** [prisma/prisma#3950] [Q&A] [answered]
I'm trying to get started with prisma. When I run prisma init projectname, I go throught the prompts, but nothing happens and I don't get any output, so I have no way of checking out what's wrong:

`/Development/graphql/prisma$ prisma init prismatut
? Set up a new Prisma server or deploy to an existing server? Use existing database
? What kind of database do you want to deploy to? PostgreSQL
? Does your database contain existing data? No
? Enter database host localhost
? Enter database port 5432
? Enter database user dbUser
? Enter database password [hidden]
? Enter database name (the database includes the schema) prisma
? Use SSL? No

~/Development/graphql/prisma$`

Is there any log I can check?

**Comment by ryands17** [comment, marked as answer]
You have installed the CLI for Prisma 1, which is an older version.

I would suggest following the quick start [here](https://www.prisma.io/docs/getting-started/quickstart-typescript) for an easy setup.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
